.github/workflows/ci.yml

```diff
@@ -13,19 +13,22 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        node-version: [12.x, 14.x, 16.x]
```

Comment:
Lets drop node 12 in a different PR (it should be major and probably update TS target)